crates/matrix-sdk-indexeddb/src/event_cache_store/serializer/types.rs

```diff
@@ -417,6 +417,23 @@ pub type IndexedEventPositionIndex = usize;
 #[derive(Debug, Serialize, Deserialize)]
 pub struct IndexedEventRelationKey(IndexedRoomId, IndexedEventId, IndexedRelationType);
 
+impl IndexedEventRelationKey {
+    /// Sets the related event field of this key. This is helpful
+    /// when searching for all events which are related to the given
+    /// event.
+    pub fn set_related_event_id(
```

Comment:
ditto, I'd name this `with_related_event_id`

Reply:
Agree - will do!

Reply:
fixup: [4799ce1a34bd2725cf75e52ff3ac6f66258d52d4](https://github.com/matrix-org/matrix-rust-sdk/pull/5414/commits/4799ce1a34bd2725cf75e52ff3ac6f66258d52d4)